test_styler_api.py

```
@@ -0,0 +1,133 @@
+    test_occasion = "business casual meeting"
+    
+    print(f"\n🧥 Testing outfit recommendation for:")
+    print(f"   User: {test_user_id}")
+    print(f"   City: {test_city}")
+    print(f"   Weather: {test_weather}")
+    print(f"   Occasion: {test_occasion}")
+    
+    # Test styler endpoint
+    print(f"\n📡 Calling styler endpoint...")
+    try:
+        response = requests.post(
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)